vagrant/config/boxes.ci.yaml

```diff
@@ -0,0 +1,17 @@
+operating_systems:
+  - centos7
+  - debian9
+  - debian10
+  - ubuntu1604
+  - ubuntu1804
+
+foreman_versions:
+  - 1.20
+  - 1.21
+  - 1.22
+  - 1.23
+  - 1.24
+  - nightly
```

Comment:
@evgeni so these don't work?

Reply:
They will, as long as they can't be miss-parsed as a float.

Reply:
@ehelms let's quote these for consistency

Reply:
Yea.. I did that for the all case but since this was a draft I got lazy elsewhere :)